pkg/plugin/plugin.go

```diff
@@ -317,17 +334,34 @@ func removeOvsPort(bridge string, portName string) error {
 func CmdDel(args *skel.CmdArgs) error {
 	logCall("DEL", args)
 
+	if args.Netns == "" {
+		panic("This should never happen, if it does, it means caller does not pass container network namespace as a parameter and therefore OVS port cleanup will not work")
+	}
+
 	if err := assertOvsAvailable(); err != nil {
 		return err
 	}
 
-	netconf, err := loadNetConf(args.StdinData)
+	envArgs, err := getEnvArgs(args.Args)
 	if err != nil {
 		return err
 	}
 
-	if args.Netns == "" {
-		panic("This should never happen, if it does, it means caller does not pass container network namespace as a parameter and therefore OVS port cleanup will not work")
+	var ovnPort string
```

Comment:
This is confusing. We get ovsPort during removal just so we can eventually use "br-int" default. We should handle the defaulting in a separate function.